docs/templating.md

```diff
@@ -671,8 +671,8 @@ minus sign (`-`) to the start or end block tag.
 {%- endfor %}
 ```
 
-The exact output of the above would be "12345". The `-%}` strips the whitespace
-right after the tag, and the `{%-` strips the whitespace right before the tag.
+The exact output of the above would be "12345". The `&#123;%-` strips the whitespace
```

Comment:
@JesseE I realise I was the one suggesting escaping special characters to prevent further issues. However it appears the escaped characters are also displayed as such. My apologies. 
Could you, "unescape" them again, and squash your commits? Then I'm okay to merge.